easybuild/easyconfigs/n/Nsight-Systems/Nsight-Systems-2023.3.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Binary'
+
+name = 'Nsight-Systems'
+version = '2023.3.1'
+
+homepage = 'https://developer.nvidia.com/nsight-systems'
+description = """NVIDIA® Nsight™ Systems is a system-wide performance analysis tool designed to visualize an
+application’s algorithm, help you select the largest opportunities to optimize, and tune to scale efficiently across
+any quantity of CPUs and GPUs in your computer; from laptops to DGX servers."""
+
+toolchain = SYSTEM
+
+# Download requires login https://developer.nvidia.com/downloads/assets/tools/secure/nsight-systems/2023_3/nsightsystems-linux-public-2023.3.1.92-3314722.run/
```

Comment:
For items that need manual download we have `download_instructions` available to give instructions that are printed to the person installing the software if the source file is not available.